src/hitApi.js

```diff
@@ -9,6 +11,19 @@ const getPizza = async () => {
   const pizzaData = await pizza.json();
   const data = pizzaData.data.recipes;
   showPizza(data);
+  itemCounter(data);
 };
 
+export const getLikes = async () => {
+  const likes = await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/8FcrK9POw5EbfAJUs4DD/likes'
```

Comment:
The code of this particular app `8FcrK9POw5EbfAJUs4DD`, we all need it in order to have access to comments and reservations, so please store it in a variable outside the function, preferable in another file, like involvmentApi.js

Reply:
Hi @nicupop729 ,

The PR is for the function to calculate and display number of items to the homepage.

I think it will be well organised if related comments come with related PR so it can be more clear.

This is just an opinion.